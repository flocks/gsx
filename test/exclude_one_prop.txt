test/samples/file.tsx:8:<Button size="large">
--
