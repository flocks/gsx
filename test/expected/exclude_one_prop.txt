test/samples/file.tsx:8:0:<Button size="large">
--
